buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1684,22 +1688,21 @@ class BeamModulePlugin implements Plugin<Project> {
         args '-c', "$pythonDir/scripts/run_expansion_services.sh stop --group_id ${project.name}"
       }
       setupTask.finalizedBy cleanupTask
+      config.startJobServer.finalizedBy config.cleanupJobServer
 
       // Task for running testcases in Java SDK
       def beamJavaTestPipelineOptions = [
-        "--runner=org.apache.beam.runners.portability.testing.TestPortableRunner",
-        "--jobServerDriver=${config.jobServerDriver}",
+        "--runner=PortableRunner",
```

Comment:
The change enabled remote execution of portable pipelines. `TestPortableRunner` provides `--jobServerDriver` just for easier testing (otherwise there needs to be a separate jobserver process to run integration tests). Validate xlang test suite already implemented the setup/shutdown script managing external processes so I modified the code to adopt the original `PortableRunner`. I didn't check but OOM shouldn't happen with the new tests here even with in-process job server drivers.